<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bothsliders</class>
 <widget class="QDialog" name="bothsliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera position control</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>240</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sliderPosition">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>20</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sliderPosition">
    <number>80</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="verlabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>57</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="horlabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>57</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>horlabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>verlabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endButton</sender>
   <signal>pressed()</signal>
   <receiver>bothsliders</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
